### DISCUSSION TITLE
Is it possible to init more than one instance?

### DISCUSSION THREAD

**Discussion by arnonrdp** [SSShooter/mind-elixir-core#138] [Q&A] [answered]
Hello everyone, 😄

I would like to know if it is possible to start more than one instance at the same time so that the maps are one below the other.

I was asked to create something like this:

![roadmap](https://user-images.githubusercontent.com/26576201/201660133-a5276a2a-5d44-435b-89cd-b57114f8a3ab.png)

But the best I could do was this:

<img width="867" alt="Captura de Tela 2022-11-14 às 09 58 49" src="https://user-images.githubusercontent.com/26576201/201666280-ae85833b-109b-446a-bcbd-4aa92d719529.png">

**Comment by SSShooter** [comment, marked as answer]
```javascript
const map1 = new MindElixir(optionsA)
map1.init(data1)
const map2 = new MindElixir(optionsB)
map2.init(data2)
const map3 = new MindElixir(optionsC)
map3.init(data3)
```

change `el` in options to mount mindmap on different element

**Reply by arnonrdp** [reply]
Really appreciate it.
Thanks
